src/utils/transformBundle.js

```diff
@@ -0,0 +1,28 @@
+export default function transformBundle ( source, transformers ) {
+	if ( typeof source === 'string' ) {
+		source = {
+			code: source,
+			map: null
+		};
+	}
+
+	return transformers.reduce( ( previous, transformer ) => {
+		let result = transformer( previous );
```

Comment:
Uglify is very rare in that it accepts an input sourcemap. Sourcemaps are bewildering; expecting people to manage that complexity creates a huge burden (worrying about the sourcemaps from your *own* transformation step is hard enough, let alone someone else's), and in practice would mean that this hook never got used for anything except Uglify.